content/en/docs/reference/terms/index.md

```diff
@@ -108,6 +108,10 @@ Passkeys are designed to be used without additional login challenges. All passke
 
 From the technical side, there are two flavors of passkeys: [synced](#synced-passkey) and [device-bound](#device-bound-passkey).
 
+## Passkey Provider
+
+An app and/or service that is responsible for storing and managing passkeys. Many operating systems include a default passkey provider, and many also support [third-party](#third-party-passkey-provider) providers.
```

Comment:
> ...include a default passkey provider...

Maybe "First-Party Passkey Provider" gets a definition too, and then we link it here?